### PR TITLE
ci: move to github actions

### PR DESCRIPTION
**Pull request by VincentMolinie**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Create automatic tests
- [ ] No automatic tests failures
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by VincentMolinie** [comment]
I just could not test if the release works 🤷

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "@semantic-release/git": "9.0.0",
     "husky": "4.2.5",
     "semantic-release": "17.3.0",
+    "semantic-release-rubygem": "^1.2.0",
```
Comment (arnaudbesnier) [inline package.json:30]: [Conventions] exact version is the way to go here:
```suggestion
    "semantic-release-rubygem": "1.2.0",
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          node-version: 12.13.0
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@v2
+        id: semantic
+        env:
+          GEM_HOST_API_KEY: ${{ secrets.GEM_HOST_API_KEY }}
```
Comment (arnaudbesnier) [inline .github/workflows/build.yml:86]: I added this missing secret in the repository

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      - name: Bundle install
+        run: |
+          bundle config path vendor/bundle
+          gem install bundler -v '1.17.3'
+          bundle install
+      - name: Test
+        timeout-minutes: 30
+        run: |
+          RAILS_ENV=test bundle exec rake test
+          RAILS_ENV=test bundle exec rake db:migrate && bundle exec rspec --color --format doc
```
Comment (arnaudbesnier) [inline .github/workflows/build.yml:64]: I can't find the `cc-test-reporter` logic to report the coverage.
Do I miss something?

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  SERVER_HOST: http://localhost:3001
+  WEBSOCKET_ENDPOINT: ws://localhost:3001/subscriptions
+  DEFAULT_DOMAIN: localhost:4200
+  BROCCOLI_PERSISTENT_FILTER_CACHE_ROOT: ./broccoli-tmp
```
Comment (arnaudbesnier) [inline .github/workflows/build.yml:13]: Are those environment variables really necessary?

**Review by arnaudbesnier — CHANGES_REQUESTED** [review]
A few things to change I think

File: .github/workflows/build.yml (hunk 3 above)
Comment (VincentMolinie) [reply .github/workflows/build.yml:64]: It's me I missed it my bad 😅

**Review by arnaudbesnier — APPROVED** [review]

**Pull request merged**